### DISCUSSION TITLE
Running Taxonomy service only

### DISCUSSION THREAD

**Discussion by coolbung** [Sunbird-Knowlg/Community#79] [Documentation Issues]
Is it possible to run the taxonomy service standalone ? 
In case search is not needed can taxonomy service be run with only Neo4J, and no other databases ?

@vinukumar-vs @vrayulu @maheshkumargangula

cc @rahul101001000

**Comment by maheshkumargangula** [comment]
@coolbung - Yes. It is possible to run the taxonomy-service standalone. It uses Neo4J, Cassandra and Redis (to enable Cache) databases.

**Reply by coolbung** [reply]
Is it possible to skip Cassandra ? What is stored in Cassandra ?

**Reply by AmiableAnil** [reply]
We are storing objectCategoryDefinition in Cassandra (**keyspace**: dev_category_store, **table**:category_definition_data). We can not skip Cassandra.

**Reply by maheshkumargangula** [reply]
@coolbung - Yes. We can run the taxonomy-service only from the Sunbird Knowlg BB. Below is the documentation to run it in a local Kubernetes cluster.
https://github.com/project-sunbird/knowledge-platform/blob/release-5.2.0/KNOWLG-SETUP.md
Cc: @vrayulu @pallakartheekreddy
